b2/sync/policy.py

```diff
@@ -67,30 +75,45 @@ def files_are_different(cls, source_file, dest_file, args):
             # Get the modification time of the latest versions
             source_mod_time = source_file.latest_version().mod_time
             dest_mod_time = dest_file.latest_version().mod_time
+            diff_mod_time = abs(source_mod_time - dest_mod_time)
 
-            # Source is newer
-            if dest_mod_time < source_mod_time:
-                return True
+            logger.debug(
+                'File %s: source time %s, dest time %s, diff %s, threshold %s, diff > threshold %s',
+                source_file.name, source_mod_time, dest_mod_time, diff_mod_time, compareThreshold,
+                diff_mod_time > compareThreshold
+            )
 
-            # Source is older
-            elif source_mod_time < dest_mod_time:
-                if args.replaceNewer:
+            if diff_mod_time > compareThreshold:
```

Comment:
this is done twice - here and in the log above. If it is done in one place and assigned to a variable, there will be no risk of someone changing one of those and forgetting about the other one.